src/homepage/howTo/imap-server.html

```diff
@@ -0,0 +1,250 @@
+                        any external service for being running.
+                    </p>
+
+                    <img src="/images/guice-jpa-architecture-overview.png">
+
+                    <p>
+                        JPA guice offers the following protocols:
+                    </p>
+
+                    <ul class="no-padding">
+                        <li><b>SMTP</b> For receiving emails</li>
+                        <li><b>IMAP</b> For reading emails</li>
+                        <li><b><a href="http://james.apache.org/server/manage-webadmin.html">WebAdmin</a></b> is a REST API allowing you to manage Apache JAMES</li>
+                    </ul>
+
+                    <p>The following protocols are also available:</p>
+
+                    <ul class="no-padding">
+                        <li><b>LMTP</b> local version of SMTP</li>
+                        <li><b>POP3</b> For reading emails. Lacks tests</li>
+                        <li><b>JMX</b> is used by a <a href="http://james.apache.org/server/manage-cli.html">command line</a> for administrating Apache James</li>
+                        <li>And <a href="http://james.apache.org/server/feature-protocols.html">more</a>...</li>
+                    </ul>
+
+                    <header class="major">
+                        <h2><b>Generation of a custom keystore</b></h2>
+                    </header>
+
+                    <p>In order to maintain a good level of privacy and security, James is relying on TLS cryptography
+                    for securing exchanges. We thus need to generate our own personal keystore. Note that this guide do not
+                    cover generating a keystore from SSL certificates. A security exception might be configured on the Mail
+                    User Agent.</p>
+
+                    <p>In order to create a keystore, please run: <code>keytool -genkey -alias james -keyalg RSA -keystore /path/to/james/conf/keystore</code>.
+                    James is configured with a default password <code>james72laBalle</code> (used to read the keystore). However, we will be overriding the
+                    configuration of the docker image, so you can be defining your own.</p>
+
+                    <header class="major">
+                        <h2><b>Starting james</b></h2>
+                    </header>
+
+                    <p>We want to override the configuration of the docker image with a volume.</p>
+
+                    <p>First let's retrieve a valid configuration:</p>
+
+                    <pre><code>$ git clone https://github.com/apache/james-project
+$ cp -rf james-project/dockerfiles/run/guice/jpa/destination/conf conf
+$ mv keystore conf/keystore</code></pre>
+
+                    <p>Modify all protocol configuration files to match your keystore password (imapserver.xml, lmtpserver.xml, managesieveserver.xml, pop3server.xml, smtpserver.xml).</p>
```

Comment:
I really think that changing default conf in docker images would be great as you would no longer require to customize configuration (and then fetch, etc)